src/server/biosecurity/grazing-field-how-separated/index.njk

```diff
@@ -0,0 +1,21 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% extends 'layouts/questions.njk' %}
+{% block heading %}
+  <h1 class="govuk-heading-l">Separating the incoming cattle</h1>
```

Comment:
weirdly, it doesn't seem to have  made any visual difference, but we've been wrapping these elsewhere in:

```html
{% block heading %}
  <div class="govuk-grid-row">
    <div class="govuk-grid-column-two-thirds">
      <h1 class="govuk-heading-l">{{ heading }}</h1>
    </div>
  </div>
{% endblock %}
```